src/coreclr/tools/aot/ILCompiler.RyuJit/Compiler/RyuJitCompilationBuilder.cs

```diff
@@ -143,29 +143,26 @@ public sealed override ICompilation ToCompilation()
             if (_resilient)
                 options |= RyuJitCompilationOptions.UseResilience;
 
-<<<<<<< HEAD
-            JitConfigProvider.Initialize(_context.Target, jitFlagBuilder.ToArray(), _ryujitOptions, _jitPath);
-            return CreateCompilation(options);
+            return CreateCompilation(options, jitFlagBuilder.ToArray(), _ryujitOptions, _jitPath);
```

Comment:
It would be more consistent to make `_ryujitOptions` and `_jitPath` protected instead of passing them as parameters.

Reply:
Thanks, done.